MovingBlocks/Terasology
# Feature: French translation update

Hello,
### Contains

This PR contains an update for the french translation:

- menu_fr.lang : was updated in Engine ressources assets folder 

### How to test

- Go to the Settings
- Change langage to french in the langages menu list
- Apply choice 
- Play the game to see translation

Thank you for your attention
Aizen93 and fortiseven
